src/protagonist/Orchestrator/Infrastructure/IIIF/IIIFAuth1Builder.cs

```diff
@@ -1,105 +1,106 @@
-
-        if (authServices.IsNullOrEmpty())
-        {
-            logger.LogWarning("Unable to get auth services for {Asset}", assetId);
-            return null;
-        }
-        
-        var authCookieService = authServices[0].ConvertToAuthCookieService(authSettings.SupportedAccessCookieProfiles,
-            authSettings.ThrowIfUnsupportedProfile);
-        if (authCookieService == null) return null;
-
-        if (authServices.Count == 1) return authCookieService;
-        
-        var id = authServices[0].Name;
-        var childServices = GetChildServices(authServices, id, assetId);
-        switch (authCookieService)
-        {
-            case AuthCookie1 authCookie1:
-                authCookie1.Service = childServices;
-                break;
-            case AuthCookie0 authCookie0:
-                authCookie0.Service = childServices;
-                break;
-        }
-
-        return authCookieService;
-    }
-
-    private List<IService> GetChildServices(List<AuthService> authServices, string id, AssetId assetId)
-    {
-        var services = new List<IService>(authServices.Count - 1);
-        foreach (var childAuthService in authServices.Skip(1))
-        {
-            var subService = childAuthService.ConvertToIIIFChildAuthService(id, authSettings.ThrowIfUnsupportedProfile);
-            if (subService != null)
-            {
-                services.Add(subService);
-            }
-            else
-            {
-                logger.LogWarning("Encountered unknown auth service profile '{Profile}' for asset {AssetId}",
-                    childAuthService.Profile, assetId);
-            }
-        }
-
-        return services;
-    }
-
-    private async Task<List<AuthService>> GetAuthServices(AssetId assetId, IEnumerable<string> rolesList,
-        CancellationToken cancellationToken)
-    {
-        var authServices = new List<AuthService>();
-        foreach (var role in rolesList)
-        {
-            cancellationToken.ThrowIfCancellationRequested();
-            authServices.AddRange(await authServicesRepository.GetAuthServicesForRole(assetId.Customer, role));
-        }
-
-        return authServices;
-    }
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Core.Collections;
+using DLCS.Core.Types;
+using DLCS.Model.Auth;
+using DLCS.Model.Auth.Entities;
+using IIIF;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Orchestrator.Settings;
+using AuthCookie0 = IIIF.Auth.V0.AuthCookieService;
+using AuthCookie1 = IIIF.Auth.V1.AuthCookieService;
+
+namespace Orchestrator.Infrastructure.IIIF;
+
+/// <summary>
+/// Implementation of <see cref="IIIIFAuthBuilder"/> for IIIF Auth 0.9 and 1.0
+/// </summary>
+/// <remarks>This is legacy and will be retired in the future</remarks>
+public class IIIFAuth1Builder : IIIIFAuthBuilder
```

Comment:
everywhere else seems to use `Auth` for v0.9/1 and `Auth2` for v2, whereas it seems there's `Auth1` for v1 - is it worth doing this here to keep consistency?

Reply:
no, this makes sense now - thanks for letting me know!